packages/feedback/src/util/sendFeedbackRequest.ts

```diff
@@ -42,72 +39,80 @@ export async function sendFeedbackRequest({
-      "release": "version@1.3",
       "request": {
-        "headers": {
-          "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
-        }
-      },
-      "sdk": {
-        "name": "sentry.javascript.react",
-        "version": "6.18.1"
+          "url": None,
+          "headers": {
+              "user-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.5 Safari/605.1.15"
+          },
       },
-      "tags": {
-        "key": "value"
+      "contexts": {
+          "feedback": {
+              "message": "test message",
+              "contact_email": "test@example.com",
+              "type": "feedback",
+          },
+          "trace": {
+              "trace_id": "4C79F60C11214EB38604F4AE0781BFB2",
+              "span_id": "FA90FDEAD5F74052",
+              "type": "trace",
+          },
+          "replay": {
+              "replay_id": "e2d42047b1c5431c8cba85ee2a8ab25d",
+          },
       },
-      "timestamp": "2023-08-31T14:10:34.954048",
-      "user": {
-        "email": "username@example.com",
-        "id": "123",
-        "ip_address": "127.0.0.1",
-        "name": "user",
-        "username": "user2270129"
-      }
     }
-  }
   */
 
-  // Prevent this data (which, if it exists, was used in earlier steps in the processing pipeline) from being sent to
-  // sentry. (Note: Our use of this property comes and goes with whatever we might be debugging, whatever hacks we may
-  // have temporarily added, etc. Even if we don't happen to be using it at some point in the future, let's not get rid
-  // of this `delete`, lest we miss putting it back in the next time the property is in use.)
-  delete feedbackEvent.sdkProcessingMetadata;
+  const envelope = createEventEnvelope(feedbackEvent, dsn, client.getOptions()._metadata, client.getOptions().tunnel);
+
+  let response: void | TransportMakeRequestResponse;
 
   try {
-    const path = 'https://sentry.io/api/0/feedback/';
-    const response = await fetch(path, {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-        Authorization: `DSN ${dsnToString(dsn)}`,
-      },
-      body: JSON.stringify(feedbackEvent),
-    });
-    if (!response.ok) {
-      return null;
+    response = await transport.send(envelope);
+  } catch (err) {
+    const error = new Error('Unable to send Feedback');
+
+    try {
+      // In case browsers don't allow this property to be writable
+      // @ts-expect-error This needs lib es2022 and newer
+      error.cause = err;
+    } catch {
+      // nothing to do
     }
+    throw error;
+  }
+
+  // TODO (v8): we can remove this guard once transport.send's type signature doesn't include void anymore
+  if (!response) {
     return response;
-  } catch (err) {
-    return null;
   }
+
+  // If the status code is invalid, we want to immediately stop & not retry
```

Comment:
IMHO we should not retry here at all - this should just fail if it fails! I would remove this overhead here, we only have it in replay because follow-up requests depend on this (and even there IMHO we should revisit if we can/should get rid of this - not sure how often that even happens!)

Reply:
Ah there's actually no retry logic, this comment is copied from replay

Reply:
ah ok, makes sense too, but then I think we can skip all this code checking the status and throwing a custom error etc, this was only needed to be able to differentiate different types of errors here!

Reply:
I think we'll still want it, as the UI has an error state to show